avm/res/insights/activity-log-alert/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -35,9 +35,7 @@ module nestedDependencies 'dependencies.bicep' = {
   scope: resourceGroup
   name: '${uniqueString(deployment().name, location)}-nestedDependencies'
   params: {
-    managedIdentityName: 'dep-${namePrefix}-msi-${serviceShort}'
     actionGroupName: 'dep-${namePrefix}-ag-${serviceShort}'
```

Comment:
I think without the 'location' parameter we'll get a lot of Linter warnings. While not mandatory - we're actually using it currently in ever single test. Hence I'd advice to keep it. We may eventually disable its need in the bicepconfig file though and then do a cleanup for all files.

Reply:
In this case, the only dependency is an action group, which is a 'global' resource and doesn't use the location parameter. Keeping it generates a linter warning `Parameter "location" is declared but never used..`. Should we keep it anyway?

Reply:
Nope, as per the other PR this was a mistake as I misjudged the loctation of the location (pun not intended)